hw2/css/style.css

```diff
@@ -0,0 +1,122 @@
+body {
+	background-color: #f9f9f9;
+	font-family: OpenSans, Ubuntu, Antic, sans-serif;
+} 
+.wrapper {
+	margin: auto;
+	padding: 50px 48px 60px 46px;
+	width: 1060px;
+	background-color: #ffffff;
+	box-shadow: 0 5px 5px rgba(0, 0, 0, 0.3);
+}
+/*HEADER*/
+.top-menu {
+	display: flex;
+	background-color: #ffffff;
+	background-image: linear-gradient(to top, #444343 0%, #5e5e5e 100%); 
+	margin-top: 40px;
+}
+.top-menu a {
+	display: block;
+	padding: 14px 25px 11px;
+	border-left: 1px solid rgba(255,255,255,0);
+    border-right: 1px solid rgba(255,255,255,0);
+	text-shadow: 1px 1px 1px rgba(0,0,0,0.5);
+	color: #ffffff;
+	font-size: 15px;
+}
+.top-menu a:hover {
+	background-color: #ffffff;
+	background-image: linear-gradient(to top, #2e2e2e 0%, #4a4a4a 100%)	;
+    border-left: 1px solid rgba(255,255,255,1);
+    border-right: 1px solid rgba(255,255,255,1);
+}
+/*CONTAINER*/
+.container {
+	padding-top: 40px;
+	padding-bottom: 60px;
+	overflow: hidden;
```

Comment:
in general use overflow hidden only in cases when you are totally sure, that there will be no popups in this container to be hidden or cut.